### PR TITLE
Upgraded fastbuild version to 0.98

### PR DESCRIPTION
**Pull request by joshuahuburn**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Upgraded Fastbuild to 0.98

#### Primary reviewers
@jared-improbable

### CODE REVIEW

File: SpatialGDK/Extras/fastbuild/install.ps1 [hunk 1]
```diff
@@ -6,7 +6,7 @@ Import-Module BitsTransfer
 Add-Type -AssemblyName System.IO.Compression.FileSystem
 
 $fileshare="\\lonv-file-01"
-$version="v0.96"
+$version="v0.98"
```
Comment (jared-improbable) [inline SpatialGDK/Extras/fastbuild/install.ps1]: What about the uninstall script? I'd copy the current 0.96 version and name it `uninstall-0.96.ps1` so people can both move from 0.96, and also uninstall/reinstall 0.98

Comment (joshuahuburn) [reply SpatialGDK/Extras/fastbuild/install.ps1]: Good idea, done.

File: SpatialGDK/Extras/fastbuild/uninstall.ps1 [hunk 2]
```diff
@@ -37,7 +37,7 @@ if ($service) {
 }
 
 # Manually remove marker file from the brokerage, since it's not possible to cleanly exit FBuildWorker.exe
-$brokerageFile = [System.IO.Path]::Combine($env:FASTBUILD_BROKERAGE_PATH, "main", "17", [System.Net.Dns]::GetHostName())
+$brokerageFile = [System.IO.Path]::Combine($env:FASTBUILD_BROKERAGE_PATH, "main", "19.windows", [System.Net.Dns]::GetHostName())
```
Comment (jared-improbable) [inline SpatialGDK/Extras/fastbuild/uninstall.ps1]: Good catch - can you add a comment to the top of the file next to the version saying what you'll need to do to keep this line up to date?

Comment (joshuahuburn) [reply SpatialGDK/Extras/fastbuild/uninstall.ps1]: Will do!

**Pull request merged**